docs/metrics.md

```diff
@@ -0,0 +1,113 @@
+# METRICS
+
+## Data Analysis
+
+The collected data will primarily be used to answer the following questions.
+Images are used for visualization and are not composed of actual data.
+
+### Do people use this?
+
+What is the overall engagement of Snooze Tabs? **This is the standard Daily
+Active User (DAU) and Monthly Active User (MAU) analysis.** This captures data
+from the people who have the add-on installed, regardless of whether they are
+actively interacting with it.
+
+### Immediate Questions
+
+TBD
```

Comment:
Also, need to formulate the questions here. Hoping we have them to be copypasta'd from a product doc somewhere. Or, I can gin them up myself if necessary because I know the best words